src/pages/Encounters/EncounterList.tsx

```diff
@@ -265,39 +274,26 @@ export function EncounterList({
                       )}
                     </Button>
                   </PopoverTrigger>
-                  <PopoverContent className="w-[320px] p-3" align="start">
+                  <PopoverContent
+                    className="w-[320px] p-3"
```

Comment:
use rem for instead of pixels; pixel arbirtrary values doesn't account for DPI settings of the display. but rem values does (handled by browser)